app/presenters/activity_presenter.rb

```diff
@@ -197,7 +197,7 @@ def oda_eligibility
   end
 
   def call_to_action(attribute)
-    if send(attribute).blank?
+    if send(attribute).nil?
```

Comment:
Would you mind adding some detail to the commit message about why this has changed? For future readers reading this commit message, a bit more context would be useful.